docs/api-reference/next/image.md

```diff
@@ -14,29 +14,41 @@ description: Enable Image Optimization with the built-in Image component.
+| `v11.1.0` | `onLoadingComplete` and `lazyBoundary` props added.                                                                                                                                                                                                                  |
+| `v11.0.0` | `src` prop support for static import.<br/>`placeholder` prop added.<br/>`blurDataURL` prop added.                                                                                                                                                                    |
+| `v10.0.5` | `loader` prop added.                                                                                                                                                                                                                                                 |
+| `v10.0.1` | `layout` prop added.                                                                                                                                                                                                                                                 |
+| `v10.0.0` | `next/image` introduced.                                                                                                                                                                                                                                             |
 
 </details>
 
-> **Note**: This page is the API reference for the `next/image` component. For a feature overview and usage information, please see the [Image Component and Image Optimization](/docs/basic-features/image-optimization.md) documentation.
+This API reference will help you understand how to use [props](#props) and [configuration options](#configuration-options) available for the Image Component. For features and usage, please see the [Image Component](/docs/basic-features/image-optimization.md) page.
 
-> **Note**: If you are using a version of Next.js prior to 13, you'll want to use the [next/legacy/image](/docs/api-reference/next/legacy/image.md) documentation since the component was renamed.
+```jsx
+import Image from 'next/image'
 
-This `next/image` component uses browser native [lazy loading](https://caniuse.com/loading-lazy-attr), which may fallback to eager loading for older browsers before Safari 15.4. When using the blur-up placeholder, older browsers before Safari 12 will fallback to empty placeholder. When using styles with `width`/`height` of `auto`, it is possible to cause [Layout Shift](https://web.dev/cls/) on older browsers before Safari 15 that don't [preserve the aspect ratio](https://caniuse.com/mdn-html_elements_img_aspect_ratio_computed_from_attributes). For more details, see [this MDN video](https://www.youtube.com/watch?v=4-d_SoCHeWE).
```

Comment:
Can we preserve some of this information? This was added because users were confused about which browsers were supported.